### ISSUE TITLE
TypeScript declaration incorrect

### ISSUE THREAD

**Issue by milesj** [epoberezkin/fast-deep-equal#9]
When using this lib I get the following TS error: "error TS2309: An export assignment cannot be used in a module with other exported elements."

**Comment by epoberezkin** [comment]
What typescript version? Why does it pass the test then I wonder...

**Comment by milesj** [comment]
Was using 2.7.*

**Comment by cwayfinder** [comment]
Currently, I'm having such an error

```
ERROR in ../node_modules/fast-deep-equal/es6/index.d.ts:1:1 - error TS1046: Top-level declarations in .d.ts files must start with either a 'declare' or 'export' modifier.
```

when importing this way

```
import * as equal from 'fast-deep-equal/es6';
```

**Comment by lincolnthree** [comment]
This is still a problem in TS 3.7 with latest version of FDE.

**Comment by lorenzofox3** [comment]
@lincolnthree do you have a workaround to get the compilation done ?

**Comment by lincolnthree** [comment]
@lorenzofox3 I copied the code into my project directly. It's just one file :)

**Comment by lincolnthree** [comment]
Just remove the unused code for things like React.

**Comment by lorenzofox3** [comment]
Thanks. For the record, I think the typing error just occurs when one requires the es6 version: 
``import equal from 'fast-deep-equal/es6 '``

**Comment by mokeev1995** [comment]
Send PR with a fix (#48), should help when it will be delivered in npm registry
